content/resources/chapter-8/building-blocks.tsx

```diff
@@ -213,6 +251,58 @@ const pythonChallengeFour = {
   ],
 }
 
+const javascriptChallengeFive = {
+  program: `console.log("KILL")`,
+  defaultFunction: {
+    name: 'verify',
+    args: [],
+  },
+  defaultCode: `const validateBlock = ()=> {
+  let fee = 0
+  let subsidy = getSubsidy(block["height"])
+
+  for (const tx of block["height"]) {
+    fee+= getTxFee(tx)
+  }
+  return subsidy + fee == block["txs"][0]["outputs"][0]["value"]
+  }
+}
```

Comment:
I am not able to proceed using this code

Reply:
lets use something like this.
I had trouble when trying to use the `block["height"]` object pointer style
``` suggestion
  defaultCode: `function validateBlock(block) {
  let subsidy = getSubsidy(block["height"]);
  let fee = 0;

  for (let i = 1; i < block["txs"].length; i++) {
    fee += getTxFee(block["txs"][i]);
  }
  
  return subsidy + fee === block["txs"][0]["outputs"][0]["value"]
}
```